components/hmpo-character-count/macro.njk

```diff
@@ -0,0 +1,7 @@
+{% macro hmpoCharacterCount(ctx, params, base) %}
```

Comment:
We do already have a hmpo-chars-left component - That I think effectively matches what this component is doing.

Appreciate that component wasn't using the govuk one but in my head we'd refactor that component over introducing a new separate component. (That would mean we need to test that components current uses i.e feedback-submit page)

Reply:
Yeah my thoughts on this are I think that we created the charsLeft before GDS had a component at all and now by having 2 components doing v similar things we're introducing a lack of consistency. 

At the very least I think we should be creating a debt stories to refactor these together and review changing feedback to new components.